applications/web3-open-jurisdiction-stack.md

```diff
@@ -0,0 +1,346 @@
+
+To give an idea, although the creation of the actual questionnaire will be part of the _Milestone 1-1_, here is an example discussion guide and it will include questions such as:
+
+Treasury research 
+
+- Do you find the process of applying to the treasury fair and just?
+- Do you feel protected in case issues do arise while working on a proposal?
+- Do you feel there should be a clear process for handling such issues?
+- Do you feel comfortable in allocating funds to entities?
+- Did it ever happen that you felt the outcome of a relationship with a treasury was unfair?
+- Do you think there is a need for a legally binding resolution method?
+- How would you currently resolve a disagreement with your counterparty?
+
+Additionally, we will conduct a comprehensive analysis of grants, focusing on their accountability and the value generated. Notably, when subjective Key Performance Indicators (KPIs) are involved, we anticipate encountering a higher frequency of points of friction.
+
+A sample discussion guide will be along the following lines:
+
+Grants research
+
+- Do you find the process of applying to grants fair and just?
+- Do you feel protected in case issues do arise while working on a grant?
+- Do you feel there should be a clear process for handling such issues?
+- Are you comfortable with the degree of trust in the Web3 Foundation involved in applying for a grant?
+
+The goal here will be to analyse in detail a minimum of 10 grants and engage with involved parties to understand how their experience was. The main focus being:
+
+- How would you rate your experience?
+- Did you feel you were dependent on your counterparty acting fairly?
+- Did you ever consider that your counterparty may not fulfil their obligations (e.g. deliver work or send a payment)?
+- How would you have behaved in such a situation?
+- Do you think a clear procedure for handling disagreements would have helped?
+- Do you think it would be better for such a procedure to have legal value in the real world?
+
+**The data collection and analysis procedures.**
+
+To facilitate data collection and collaboration, treasury and grants data will be documented in a Google spreadsheet. This spreadsheet will predominantly contain publicly available data but will also include anonymized opinions from stakeholders regarding the effectiveness of specific grants.
+
+Throughout the discussion guide we aim to leverage NPS and similar metrics to be able to extract readable graphs that can inform the subsequent development of the stack and also to be publicly available references for the community of the work done.
```

Comment:
```suggestion
Throughout the discussion guide we aim to leverage NPS (Net Promoter Score) and similar metrics to be able to extract readable graphs that can inform the subsequent development of the stack and also to be publicly available references for the community of the work done.
```

I just thought it'd be easier to read if this acronym would be explained at its first appearance (rather than 2nd).